content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,724 @@
+---
+reviewers:
+  - sftim
```

Comment:
```suggestion
```
I don't have any special ConfigMap skills, so I wouldn't list me here.

Reply:
I've handled this in the latest commit.